Dockerfile

```diff
@@ -1,7 +1,6 @@
-FROM python:3.6-slim-buster
-ENV PYTHONUNBUFFERED True
+FROM tiangolo/meinheld-gunicorn-flask:python3.8-alpine3.11
 WORKDIR /app
-ADD . /app
-RUN pip install -r requirements.txt
-EXPOSE 5000
-CMD ["waitress-serve","--port=5000", "comp2env:app"]
+COPY . /app
+RUN apk add gcc libc-dev g++ libffi-dev libxml2 unixodbc-dev postgresql-dev; \
```

Comment:
Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`
Use the `--no-cache` switch to avoid the need to use `--update` and remove `/var/cache/apk/*` when done installing packages
Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>`